pom.xml

```diff
@@ -167,12 +167,12 @@
         <dependency>
             <groupId>fr.acinq.secp256k1</groupId>
             <artifactId>secp256k1-kmp-jvm</artifactId>
-            <version>0.5.1</version>
+            <version>0.6.2</version>
         </dependency>
         <dependency>
             <groupId>fr.acinq.secp256k1</groupId>
             <artifactId>secp256k1-kmp-jni-jvm</artifactId>
-            <version>0.5.1</version>
+            <version>0.6.2</version>
         </dependency>
         <dependency>
             <groupId>org.jetbrains.kotlin</groupId>
```

Comment:
the `kotlin-stdlib-jdk8` dependency should be removed, so users of the library will use the one specified by `secp256k1-kmp`

Reply:
Done